### PR TITLE
Remove NUnit Software from Copyright; add MIT license.

### LINKED ISSUE #136: Resolve conflicting Copyright statements
@CharliePoole commented on [Sun Nov 13 2016](https://github.com/nunit/nunit3-vs-adapter/issues/265)

For historical reasons, we state that the copyright for the adapter is held by Charlie Poole in some places and by NUnit Software in others. It's not too terrible, since NUnit Software is a DBA of mine, but it's potentially confusing and should be resolved.

There is no good category for this, so I'm just calling it a build issue. Priority is high because I want the current copyright to be completely clear in order to be able to easily transfer it elsewhere at some point.

### PR DESCRIPTION
**Pull request by CharliePoole**
Fixes #136 

**Only @CharliePoole or @OsirisTerje should merge**
We are still discussing this

### CODE REVIEW

File: nuget/NUnitVisualStudioTestAdapter.nuspec [hunk 1]
```diff
@@ -15,7 +15,7 @@ Note that this package ONLY contains the adapter, not the NUnit framework.  You
 You only need one such package for a solution. The package works with Visual Studio >= 2012</description>
       <summary>The NUnit TestAdapter for Visual Studio 2012/13/15 for NUnit 2 and lower</summary>
```
Comment (OsirisTerje) [inline nuget/NUnitVisualStudioTestAdapter.nuspec]: We should add "/17" to this summary text, while at it.

File: src/NUnitTestAdapter/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -12,7 +12,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("NUnit Software")]
 [assembly: AssemblyProduct("NUnit.VisualStudio.TestAdapter")]
-[assembly: AssemblyCopyright("Copyright 2011-2017, NUnit Software")]
+[assembly: AssemblyCopyright("Copyright (c) 2011-2015, Charlie Poole")]
```
Comment (OsirisTerje) [inline src/NUnitTestAdapter/Properties/AssemblyInfo.cs]: The previous was "-2017", shouldn't the new one also be the same?

File: nuget/NUnitVisualStudioTestAdapter.nuspec (hunk 1 above)
Comment (CharliePoole) [inline nuget/NUnitVisualStudioTestAdapter.nuspec]: Good call.

File: src/NUnitTestAdapter/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (CharliePoole) [inline src/NUnitTestAdapter/Properties/AssemblyInfo.cs]: Merge error fixing conflicts.

**Comment by CharliePoole** [comment]
What we have generally done in the past is update the date in the source when that file has a significant change. I'll do another commit applying your comments and now adding your name as discussed.

File: nuget/NUnitVisualStudioTestAdapter.nuspec (hunk 1 above)
Comment (CharliePoole) [inline nuget/NUnitVisualStudioTestAdapter.nuspec]: Also, removing "and lower" now that I noticed it. NUnit 2 is as low as we go.

**Comment by CharliePoole** [comment]
OK, I think this is now ready.

File: nuget/NUnitVisualStudioTestAdapter.nuspec [hunk 3]
```diff
@@ -4,18 +4,18 @@
...
         <iconUrl>http://nunit.org/nuget/nunit_32x32.png</iconUrl>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <description>The NUnit TestAdapter for Visual Studio 2012/13/15 for NUnit 2.  With this package you don't need to install the VSIX adapter package, and you don't need to upload the adapter to your TFS server.
 Note that this package ONLY contains the adapter, not the NUnit framework.  You must also get the framework, or use the NUnitTestAdapter.WithFramework nuget package.
 You only need one such package for a solution. The package works with Visual Studio >= 2012</description>
-      <summary>The NUnit TestAdapter for Visual Studio 2012/13/15 for NUnit 2 and lower</summary>
+      <summary>The NUnit TestAdapter for Visual Studio 2012/13/15/17 for NUnit 2</summary>
         <releaseNotes>Hotfix for .net framework 3.5 support, and missing pdb crash</releaseNotes>
-        <copyright>Copyright (c) Charlie Poole 2011-2016</copyright>
+        <copyright>Copyright (c) Charlie Poole 2011-20017, Terje Sandstrom 2014-2017</copyright>
```
Comment (jnm2) [inline nuget/NUnitVisualStudioTestAdapter.nuspec]: `20017`

File: src/NUnitTestAdapter/NUnitEventListener.cs [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ﻿// ***********************************************************************
-// Copyright (c) 2011 Charlie Poole
+// Copyright (c) 2011 Charlie , Terje Sandstrom
```
Comment (jnm2) [inline src/NUnitTestAdapter/NUnitEventListener.cs]: Missing <code>&nbsp;Poole</code>

File: src/NUnitTestAdapter/Properties/AssemblyInfo.cs [hunk 5]
```diff
@@ -12,7 +12,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("NUnit Software")]
 [assembly: AssemblyProduct("NUnit.VisualStudio.TestAdapter")]
-[assembly: AssemblyCopyright("Copyright (c) 2011-2015, Charlie Poole")]
+[assembly: AssemblyCopyright("Copyright (c) 2011-201 7 Charlie Poole, 2014-2017 Terje Sandstrom")]
```
Comment (jnm2) [inline src/NUnitTestAdapter/Properties/AssemblyInfo.cs]: `201 7`

**Comment by CharliePoole** [comment]
Thanks @jnm2! Lots of bugs in my global replacement software. 😈

**Comment by CharliePoole** [comment]
@OsirisTerje I made a judgement to just leave a single year in each file (last year a major change was made) and to do the full range of years for each of us in the top-level copyright statements.

@nunit/core-team This PR of mine can be considered as my assigning shared copyright over the nunit-vs-adapter software to Terje, recognizing the scope of his contributions. If it should be necessary at some point, I'll do something more formal.

**Comment by CharliePoole** [comment]
@OsirisTerje Does this look OK to you now?

**Pull request merged**